### PR TITLE
Configure auth0 and use it for comments

### PR DESCRIPTION
**Pull request by sergous**
https://github.com/dtpstat/dtp-map/issues/80

## User profile page - before Login
Снимок экрана 2021-12-15 в 3 41 49 AM

## User profile page - after Login
Снимок экрана 2021-12-15 в 3 41 17 AM

## User comments - before Login
Снимок экрана 2021-12-15 в 3 42 36 AM

## User comments - after Login
Снимок экрана 2021-12-15 в 2 44 52 AM

### CODE REVIEW

File: src/pages/iframes/comments/[dtpId].tsx [hunk 1]
```diff
@@ -22,15 +23,17 @@ const CommentsIframePage: NextPage<CommentsIframePageProps> = ({
   }
 
   return (
-    <CommentsProvider initComments={comments}>
-      <CommentList />
+    <UserProvider>
```
Comment (kachkaev) [inline src/pages/iframes/comments/[dtpId].tsx:26]: We can probably move this into `_app.tsx`

File: README.md [hunk 2]
```diff
@@ -17,3 +17,17 @@ Open [http://localhost:3000](http://localhost:3000) with your browser to see the
...
 ```
+
+## Authorization configuration
+
+Create file **.env.local** and fill it with information from [Auth0 Application Profile](https://manage.auth0.com/dashboard):
+
+```ini
+AUTH0_SECRET=replace-with-your-own-secret-generated-with-openssl
+AUTH0_BASE_URL=http://localhost:3000
+AUTH0_ISSUER_BASE_URL='https://{DOMAIN}'
```
Comment (kachkaev) [inline README.md:28]: Any way we can configure auth0 for Netlify preview releases? I wonder if there is some recipe we can find.

**Review by kachkaev — APPROVED** [review]
🎉 Looking nice!

File: README.md (hunk 2 above)
Comment (kachkaev) [reply README.md:28]: Meanwhile, I’m getting this when trying to press login at  https://deploy-preview-7--dtp-stat-on-nextjs.netlify.app/

File: src/pages/iframes/comments/[dtpId].tsx (hunk 1 above)
Comment (sergous) [reply src/pages/iframes/comments/[dtpId].tsx]: Done

File: README.md (hunk 2 above)
Comment (sergous) [reply README.md:28]: Found and tested

**Pull request merged**